sjb/config/test_cases/ami_build_origin_int_rhel_build.yml

```diff
@@ -70,6 +70,15 @@ actions:
       git log -1 --pretty=%h >> "${metrics_repo}/ORIGIN_COMMIT"
       console_repo="/data/src/github.com/openshift/origin-web-console-server/"
       git log -1 --pretty=%h >> "${console_repo}/ORIGIN_COMMIT"
+  - type: "script"
+    title: "build the builder container image"
+    repository: "builder"
+    timeout: 3600
+    script: |-
+      if [[ "${PULL_BASE_REF}" == "master" ]]; then
```

Comment:
not sure if this is right (other jobs did it, but this job did not seem to do it for the images it builds)